pkg/datapath/linux/node.go

```diff
@@ -227,7 +227,7 @@ func createDirectRouteSpec(CIDR *cidr.CIDR, nodeIP net.IP) (routeSpec *netlink.R
 
 	routes, err = netlink.RouteGet(nodeIP)
 	if err != nil {
-		err = fmt.Errorf("unable to lookup route for node %s: %s", nodeIP, err)
+		err = fmt.Errorf("unable to lookup route for node %s: %w", nodeIP, err)
```

Comment:
This change should be squashed in commit c076ddcfd45ca763f28914caab68c7596f34bc77